README.md

```diff
@@ -1,37 +1,168 @@
 # splinepy
 [![workflow](https://github.com/tataratat/splinepy/actions/workflows/main.yml/badge.svg)](https://github.com/tataratat/splinepy/actions)
```

Comment:
Can this line be deleted? I consider this as internal

Reply:
this is deploy and full build status for, thought it's standard

Reply:
its just a hyperlink. there is no nice image

Reply:
there was a mismatch in naming. it works now and now I am gonna rename the workflow real quick